javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,95 @@
+// Voice assistant
+
+let name = "";
+let todo = [];
+function getReply(command) {
+  let phraseArr = command.toLowerCase().split(" ");
+
+  if (
+    phraseArr[phraseArr.indexOf("is")] &&
+    phraseArr[phraseArr.indexOf("is") + 1] &&
+    !phraseArr.includes("what")
+  ) {
+    name = phraseArr.slice(phraseArr.indexOf("is") + 1, phraseArr.length);
+    console.log("nice to meet you " + name.join(" "));
+  } else if (
+    phraseArr.includes("what") &&
+    phraseArr.includes("name") &&
+    name !== ""
+  ) {
+    console.log("your name is " + name.join(" "));
+  } else if (phraseArr.includes("add") && phraseArr.includes("todo")) {
+    let todoTask = phraseArr
+      .slice(phraseArr.indexOf("add") + 1, phraseArr.indexOf("to"))
+      .join(" ");
+    if (!todoTask) {
+      console.log("please enter some todo !!");
+    } else {
+      todo.push(
+        phraseArr
+          .slice(phraseArr.indexOf("add") + 1, phraseArr.indexOf("to"))
+          .join(" ")
+      );
+    }
+  } else if (phraseArr.includes("remove") && phraseArr.includes("todo")) {
+    let item = phraseArr
+      .slice(phraseArr.indexOf("remove") + 1, phraseArr.indexOf("from"))
+      .join(" ");
+    todo.splice(todo.indexOf(item), 1);
+  } else if (phraseArr.includes("what") && phraseArr.includes("todo")) {
+    console.log("you have " + todo.length + " todo " + todo.join(" and "));
```

Comment:
This is good, but try to use template literals as its a new feature added in ES6 and makes it easier to read and write

Reply:
Honestly i couldn't find it in my keyboard :flushed: , i will try it again by copy past